dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/RpcServerContextAttachment.java

```diff
@@ -0,0 +1,138 @@
+            return RpcContext.getServerResponseContext().stopAsync();
+        }
+
+        @Override
+        public AsyncContext getAsyncContext() {
+            return RpcContext.getClientResponseContext().getAsyncContext();
+        }
+
+        @Override
+        public String getAttachment(String key) {
+            return RpcContext.getClientResponseContext().getAttachment(key);
+        }
+
+        @Override
+        public Object getObjectAttachment(String key) {
+            return RpcContext.getClientResponseContext().getObjectAttachment(key);
+        }
+
+        @Override
+        public RpcContextAttachment setAttachment(String key, String value) {
+            return RpcContext.getServerResponseContext().setAttachment(key, value);
+        }
+
+        @Override
+        public RpcContextAttachment setAttachment(String key, Object value) {
+            return RpcContext.getServerResponseContext().setAttachment(key, value);
+        }
+
+        @Override
+        public RpcContextAttachment removeAttachment(String key) {
+            return RpcContext.getServerResponseContext().removeAttachment(key);
+        }
+
+        @Override
+        public Map<String, String> getAttachments() {
+            return RpcContext.getServerResponseContext().getAttachments();
+        }
+
+        @Override
+        public Map<String, Object> getObjectAttachments() {
+            return RpcContext.getServerResponseContext().getObjectAttachments();
+        }
+
+        @Override
+        public RpcContextAttachment setAttachments(Map<String, String> attachment) {
+            return RpcContext.getServerResponseContext().setAttachments(attachment);
+        }
+
+        @Override
+        public RpcContextAttachment setObjectAttachments(Map<String, Object> attachment) {
+            return RpcContext.getServerResponseContext().setObjectAttachments(attachment);
+        }
+
+        @Override
+        public void clearAttachments() {
+            RpcContext.getServerResponseContext().clearAttachments();
+        }
+
+        @Override
+        public Map<String, Object> get() {
+            return RpcContext.getServerResponseContext().get();
+        }
+
+        @Override
+        public RpcContextAttachment set(String key, Object value) {
+            return RpcContext.getServerResponseContext().set(key, value);
+        }
+
+        @Override
+        public RpcContextAttachment remove(String key) {
+            return RpcContext.getServerResponseContext().remove(key);
+        }
+
+        @Override
+        public Object get(String key) {
+            return RpcContext.getServerResponseContext().get(key);
+        }
+
+        @Override
+        public RpcContextAttachment copyOf(boolean needCopy) {
+            return RpcContext.getServerResponseContext().copyOf(needCopy);
+        }
+
+        @Override
+        protected boolean isValid() {
+            return RpcContext.getServerResponseContext().isValid();
```

Comment:
Internal method, should not be called. Throw exception directly. BTW, check the usage.